_inc/client/plans/single-product-backup.jsx

```diff
@@ -13,6 +15,61 @@ import ExternalLink from 'components/external-link';
 
 import './single-product-backup.scss';
 
+export function SingleProductBackup( { products, upgradeLinks } ) {
+	const [ selectedBackupType, setSelectedBackupType ] = useState( 'real-time' );
+
+	const backupPlanPrices = {
+		jetpack_backup_daily: {
+			monthly: get( products, [ 'jetpack_backup_daily_monthly', 'cost' ], '' ),
+			yearly: get( products, [ 'jetpack_backup_daily', 'cost' ], '' ),
+		},
+		jetpack_backup_realtime: {
+			monthly: get( products, [ 'jetpack_backup_realtime_monthly', 'cost' ], '' ),
+			yearly: get( products, [ 'jetpack_backup_realtime', 'cost' ], '' ),
+		},
+	};
+
+	const currency_code = get( products, [ 'jetpack_backup_daily', 'currency_code' ], '' );
+	const currencySymbol = getCurrencyDefaults( currency_code ).symbol;
+
+	return (
+		<React.Fragment>
+			<h1 className="plans-section__header">{ __( 'Solutions' ) }</h1>
+			<h2 className="plans-section__subheader">
+				{ __( "Just looking for backups? We've got you covered." ) }
+			</h2>
+			<div className="single-product-backup__accented-card-container">
+				<div className="single-product-backup__accented-card">
+					<div className="single-product-backup__accented-card__header">
+						<SingleProductBackupHeader
+							currencySymbol={ currencySymbol }
+							backupPlanPrices={ backupPlanPrices }
+						/>
+					</div>
+					<div className="single-product-backup__accented-card__body">
```

Comment:
I'm not a fun of CSS sub-elements like here. We could just flatten the class name to something like:

```html
<div className="single-product-backup__accented-card-body">
```